### DISCUSSION TITLE
Grafana Unified Alerting - multidimensionality + screenshots in emails

### DISCUSSION THREAD

**Discussion by gg-g** [grafana/grafana#56525] [Alerting]
I have a dashboard with repeating rows for a variable. The row contains a panel on which I defined an alert using Grafana unified alerting. I’m using multi-dimensional alerting to create multiple alert instances. The notifications are sent by email. All of this works fine.

When testing and triggering alerts for several instances of the variable, I noticed that in each email the same panel screenshot (first on the page for the first variable instance) is included even though the labels and values correspond to other variable instances.

Is this a limitation?

The only information I found that could be related was on [Images in notifications - Limitations](https://grafana.com/docs/grafana/latest/alerting/images-in-notifications/#limitations) where it states:

>     Some contact points only handle a single image. In this case, the first image associated with an alert will be attached.

Is this the case here? 

**What you expected to happen**:

To have the correct panel/per email.

**How to reproduce it (as minimally and precisely as possible)**:

- Set up dashboard with variable and use variable for repeating rows
- Set up alert with email notification in new alerting and making sure multi-dimensional alerts are generated based on the column we also get our dashboard variable from

**Anything else we need to know?**:

**Environment**:
- Grafana version: 9.1.6 (92461d8d1e)
- Data source type & version: MySQL (MariaDB 10.6.7)
- OS Grafana is installed on: Red Hat Enterprise Linux Server release 7.9 (Maipo)
- User OS & Browser: Windows 10 & Firefox 102.3.0esr (64-bit)
- Grafana plugins:
    - Grafana Image Renderer version: 3.6.1

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
